config.schema.json

```diff
@@ -63,12 +63,6 @@
         "default": true,
         "required": true
       },
-      "delay": {
-        "title": "Startdelay (beta) just for testing",
-        "type": "boolean",
-        "default": false,
-        "required": false
-      },
```

Comment:
This was deleted in `0.17.0` but forgot to delete it in the schema. Maybe it's worth applying this change to `master`